universal-application-tool-0.0.1/app/filters/LoggingFilter.java

```diff
@@ -0,0 +1,43 @@
+package filters;
+
+import java.time.Clock;
+import java.util.concurrent.Executor;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import play.mvc.EssentialAction;
+import play.mvc.EssentialFilter;
+
+/** This is a simple filter that produces an apache-style access log. */
+@Singleton
+public class LoggingFilter extends EssentialFilter {
+
+  private final Executor exec;
+  private final Clock clock;
+  private static final Logger log = LoggerFactory.getLogger("loggingfilter");
```

Comment:
**Question**: Is this (`loggingfilter`) a specific name (name of a library or a component)? Or is it defined by you in `logback.xml`?

Reply:
It's a reference to `logback.xml`, yep!  Any string is acceptable, logback configures where the logs go by name.